metab_import.py

```diff
@@ -524,33 +531,44 @@ def main():
 
     # Projects
     projects = get_projects(mwb_cur, sup_cur, people, orgs)
-    project_triples, project_summaries = make_projects(aide.namespace, projects)
+    project_triples, project_summaries = \
+        make_projects(aide.namespace, projects)
     all_proj_triples = project_triples + project_summaries
     print_to_file(all_proj_triples, project_file)
 
     # Studies
     # Study file printed after datasets
     studies = get_studies(mwb_cur, sup_cur, people, orgs)
-    study_triples, study_summaries = make_studies(aide.namespace, studies, projects)
+    study_triples, study_summaries = \
+        make_studies(aide.namespace, studies, projects)
 
     # Datasets
     datasets = get_datasets(mwb_cur)
-    dataset_triples, study_sup_triples = make_datasets(aide.namespace, datasets, studies)
+    dataset_triples, study_sup_triples = \
+        make_datasets(aide.namespace, datasets, studies)
     print_to_file(dataset_triples, dataset_file)
 
     all_study_triples = study_triples + study_summaries + study_sup_triples
     print_to_file(all_study_triples, study_file)
 
     summary_triples = project_summaries + study_summaries
-    # If you've made it this far, it's time to delete
-    aide.do_delete()
-    do_upload(aide, org_triples)
-    do_upload(aide, people_triples)
-    do_upload(aide, project_triples)
-    do_upload(aide, study_triples)
-    do_upload(aide, dataset_triples)
-    do_upload(aide, tools_triples)
-    do_upload(aide, summary_triples, 1)
+    if not DRY_RUN:
```

Comment:
Can you invert this logic?
```suggestion
    if dry_run:
        return
```